server/voters.html

```diff
@@ -0,0 +1,49 @@
+<script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
```

Comment:
Can we keep website without jquery ? Look like everybody can't do js without jquery :(